SmartDeviceLink/SDLIAPSession.m

```diff
@@ -9,17 +9,17 @@
 #import "SDLMutableDataQueue.h"
 
 NS_ASSUME_NONNULL_BEGIN
-#define IO_STREAMTHREAD_NAME            @ "com.smartdevicelink.iostream"
-#define STREAM_THREAD_WAIT_SECS         1.0
+NSString *const ioStreamThreadName  = @ "com.smartdevicelink.iostream";
+NSTimeInterval const streamThreadWaitSecs = 1.0;
 
 @interface SDLIAPSession ()
 
 @property (assign, nonatomic) BOOL isInputStreamOpen;
```

Comment:
I'm not saying we don't need to check the state of both, I'm just saying that this bool is the same as the above, and we only use it in one spot, and because of this I don't feel as if it is necessary.